docs/code-style.md

```diff
@@ -0,0 +1,257 @@
+- Smaller Widgets will help us to reuse them in different parts of the code.
+- A Widget would have the resources that it needs, and this one would be the only who can access to the resources it is using (constants, colors, strings, ...).
+
+## Assets / Images separation
+
+Inside the assets folder, we should separate the different resources by folders that identifies the place where this is being used.
+
+## Literals & magic numbers
+
+We would not have hardcoded literals or numbers in our code, that adds complexity when we try to change any of them, or if we want tu reuse them. we should place them in the correct Strings or Constants file or in the top of the current class if they are only used there.
+
+```
+class XYZ {
+
+    void example() {
+        runMethod1('current position');
+        for(int i=0; i<3; i++) {
+            runMethod1('current position $i');
+        }
+    }
+
+    void runMethod1(String text) {
+        print(text);
+    }
+
+}
+```
+
+```
+class XYZ {
+
+    void example() {
+        const currentPosition = 'current position';
+        const numberOfPositions = 3;
+        runMethod1(currentPosition);
+        for(int i=0; i<numberOfPositions; i++) {
+            runMethod1('$currentPosition $i');
+        }
+    }
+
+    void runMethod1(String text) {
+        print(text);
+    }
+
+}
+```
+## Resources
+
+Here are some resources that helped to define the code style for this project:
+- [Effective Dart](https://dart.dev/guides/language/effective-dart/style)
+- [Style Guide for Flutter](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo)
+- [Type System](https://dart.dev/guides/language/sound-dart)
+
+# Pull Request Template
+
+*When we create a new Pull Request (PR) in develop we should follow the next template:*
+
+
+[FARM-XXX - Please use Spring JIRA ticket and not subtasks]
+
+#### 📲 What
+
+A description of the change.
+
+#### 🤔 Why
+		
+Why it's needed, background context.
+		
+#### 🛠 How
+		
+More in-depth discussion of the change or implementation.
+
+#### 👀 See
+		
+Screenshots / external resources
+		 
+#### ✅ Acceptance criteria
+
+- [ ] Design Review for UI with BA completed. 
+- [ ] Manually tested and verified on Android device/emulator (min. Lollipop 5.1)
+- [ ] Showcase video / automation test
+- [ ] Passing all tests
+- [ ] Rebased/merged with latest changes from development and re-tested
+- [ ] Removed unsed comments. TODOs must have JIRA for future work.
+
+#### Coding Standards Checklist
+- [ ] unit tests 80% coverage on testable code (functions, methods, class)
+- [ ] (Architecture) redux state, reducers, actions, middleware defined under lib/redux
+- [ ] ui elements defined under lib/ui. 
+- [ ] Strings ready for localisation (e.g. defined in lib/utils/strings.dart)
+- [ ] Assets (images/icons path) defined in lib/utils/assets.dart
+- [ ] Colors defined in lib/utils/colors.dart
```

Comment:
remove this, we have style injection and then app specific styles will be in an app scope not a global scoped generic file.